odl/contrib/datasets/ct/mayo.py

```diff
@@ -126,30 +123,39 @@ def load_projections(folder, indices=None):
     src_radius = datasets[0].DetectorFocalCenterRadialDistance
     det_radius = (datasets[0].ConstantRadialDistance -
                   datasets[0].DetectorFocalCenterRadialDistance)
-
-    # Convert pitch and offset to odl defintions
     pitch = (pixel_size[1] * shape[1] * datasets[0].SpiralPitchFactor *
              src_radius / (src_radius + det_radius))
-    offset_along_axis = (datasets[0].DetectorFocalCenterAxialPosition -
-                         angles[0] / (2 * np.pi) * pitch)
 
     # Get flying focal spot data
     offset_axial = np.array([d.SourceAxialPositionShift for d in datasets])
     offset_angular = np.array([d.SourceAngularPositionShift for d in datasets])
     offset_radial = np.array([d.SourceRadialDistanceShift for d in datasets])
 
-    angles_offset = angles - offset_angular
-    src_rad_offset = src_radius + offset_radial
-    offset_x = (np.cos(angles_offset) * (-src_rad_offset) -
-                np.cos(angles) * (-src_radius))
-    offset_y = (np.sin(angles_offset) * (-src_rad_offset) -
-                np.sin(angles) * (-src_radius))
-    offset_z = offset_axial
-
-    # TODO: WE CURRENTLY IGNORE THE OFFSETS DUE TO FLYING FOCAL SPOT
-    source_offsets = np.array([offset_x, offset_y, offset_z]).T
+    if 1:
```

Comment:
I guess this is for development only?
If not, I'm not in favor of "parking" backup code like this. Better make an issue and post the old code there for reference. It's also in the git history anyway.

Reply:
I'll remove it.